pkg/apis/camel/v1/maven_types.go

```diff
@@ -32,3 +32,10 @@ type RepositoryPolicy struct {
 	UpdatePolicy   string `xml:"updatePolicy,omitempty" json:"updatePolicy,omitempty"`
 	ChecksumPolicy string `xml:"checksumPolicy,omitempty" json:"checksumPolicy,omitempty"`
 }
+
+// Build Extension
+type Extension struct {
```

Comment:
Can't we just embed the existing `MavenArtifact` struct?

Reply:
Thanks for reviewing ! Done